plugins/woocommerce/tests/e2e-pw/utils/site.js

```diff
@@ -0,0 +1,242 @@
+const api = require( './api' );
```

Comment:
File containing utility functions to reset the site to a somewhat pristine state.